app/controllers/sessions_controller.rb

```diff
@@ -1,16 +1,28 @@
 class SessionsController < ApplicationController
-  def new; end
+  skip_before_action :authorised
+
+  def new
+    p "params when new is called #{params}"
```

Comment:
Is this still needed ?

Reply:
No i was debugging something and forgot to remove, but will delete it now, thanks!